### ISSUE TITLE
Looks like macOS M1 Monterey app may need to be signed?

### ISSUE THREAD

**Issue by kyle-github** [jkoplo/CrudeObservatory#3]
See attached screenshot.

<img width="272" alt="image" src="https://user-images.githubusercontent.com/1820043/148719987-85ffa586-24d0-492f-91c4-a26c84fde4f3.png">

I am still a macOS noob, so I am not sure of the cause of this.  I tried a few things I found on line but they did not work.   Still researching.  I think it is a signing problem?

**Comment by jkoplo** [comment]
Yeah it seems like there are ways to turn off security and run non-signed apps.

If wanted to sign the app, I'd need to get an Apple developer account and pay $99 annually for the privilege. There's some indication they may waive that fee for "non-profit" apps, but you have to request a waiver.
